### PR TITLE
Set GitHub package

### PR DESCRIPTION
**Pull request by ricardoperez**
* Added github packages support
* Bump version to 6.5.1

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# [6.5.1] / 2020-06-19
```
Comment (PatrykF3D) [inline CHANGELOG.md:1]: https://semver.org

> MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards compatible manner, and
PATCH version when you make backwards compatible bug fixes.

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-6.5.0
+6.5.1
```
Comment (PatrykF3D) [inline VERSION:1]: https://semver.org

> MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards compatible manner, and
PATCH version when you make backwards compatible bug fixes.

File: license_finder.gemspec [hunk 3]
```diff
@@ -30,10 +30,16 @@ Gem::Specification.new do |s|
     'Jeff Jun'
   ]
 
-  s.email       = ['labs-commoncode@pivotal.io']
-  s.homepage    = 'https://github.com/pivotal/LicenseFinder'
+  s.email       = ['developers@forward3d.com']
+  s.homepage    = 'https://github.com/forwar3d/LicenseFinder'
```
Comment (PatrykF3D) [inline license_finder.gemspec:34]: forward3d

**Comment by ricardoperez** [comment]
> 1. I think the version bump is not sufficient
> 2. The version bump and gh packages config should not be in the same PR
> 3. Let's keep the gh packages config on a separate branch for now, we definitely don't want to open PR to original repo with that config.

So. I'm going to close this PR and open one for version bump.

**Pull request closed without merging**